src/components/BriefList/BriefsList.tsx

```diff
@@ -1,24 +1,55 @@
 import React, { useEffect, useState } from 'react';
 import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
 import { getBriefList } from '../../services/BriefDataService';
 import { IBrief } from '../../interfaces/Brief';
 import Brief from '../Brief/Brief';
 import { styles } from './styles';
+import ActionBar from '../ActionBar/ActionBar';
+import { BriefStatusesConfig } from '../../constants/BriefStatuses';
+import { BriefDatesConfig } from '../../constants/BriefDates';
+import { getFilteredBriefList } from '../../services/BriefService';
 
 const BriefList: React.FC = () => {
   const [briefList, setBriefList] = useState<IBrief[]>([]);
+  const [filteredBriefList, setFilteredBriefList] = useState<IBrief[]>([]);
+  const [statusFilter, setStatusFilter] = useState(BriefStatusesConfig.all.value);
+  const [dateSort, setDateSort] = useState(BriefDatesConfig.createdAt.value);
 
   useEffect(() => {
     getBriefList()
-      .then((response: IBrief[]) => setBriefList(response));
+      .then((response: IBrief[]) => {
+        setBriefList(response);
+        setFilteredBriefList(getFilteredBriefList(response, statusFilter));
+      });
   }, []);
```

Comment:
this effet most likely will not work as you want. It doesn't have `statusFilter` as a dependency. This means that whenever it is fired, `statusFilter` will have the initial value, as the hook is not recreated on  `statusFilter` change. 

There are actually an eslint rules that will notify you about all needed dependencies


```
"rules": {
    "react-hooks/rules-of-hooks": "error",
    "react-hooks/exhaustive-deps": "warn"
  }
```